tk_toolchain/cmd_line_tools/tk_docs_generation/sphinx_data/conf.py

```diff
@@ -366,7 +366,6 @@ def setup(app):
 # and extrnal libs
 intersphinx_mapping = {
     "python": ("https://docs.python.org/3.9", None),
-    "PySide": ("http://pyside.github.io/docs/pyside/", None),
```

Comment:
Was this variable not used? Otherwise, where is it removed?

Reply:
We're building Sphinx documentation using PySide2. That's the reason you'll see pages like [this one](https://developers.shotgridsoftware.com/tk-framework-qtwidgets/spinner_widget.html) referencing PySide2 classes.

However, I agree to keep this line for documentation legacy purposes. It won't do any harm.

Reply:
My mistake, pages like [this one](https://developers.shotgridsoftware.com/tk-framework-qtwidgets/shotgun_fields.html#shotgun_fields.ShotgunFieldManager) still references PySide (1). Line should not be deleted.